styling/Phoenix_files/_hero.scss

```diff
@@ -84,3 +85,25 @@ body, h1, h2, h3, h4, h5, h6 {
   background-image: url("../Phoenix_files/phoenix-logo-white.png") !important;
   background-size: 250px 34px;
 }
+
+#subheader-columns {
+  margin: 50px 30px 0; }
+
+.subheader h4 {
+  color: #000;
+  font-weight: $light-font;
+  font-size: 18px;
+  padding-bottom: 14px;
+  text-transform: uppercase; }
+
+.subheader p {
+  font-weight: $light-font;
+  color: #333;
```

Comment:
Yeah that's what I always do. Makes it easy to see all the colors in use and makes it easy to see if we're using too many different colors. I like those names :+1: